docs/services/data/_index.md

```diff
@@ -43,56 +43,58 @@ Or learn more about Viam's data management features, which can be broken down in
 
 ## On-device data management
 
-Manage data directly on your robot, with configurable data capture from various components, automated recovery after interruptions, and secure cloud synchronization for efficient data storage and management.
+The data management service:
+
+- Automatically captures data at the rate you specify.
+  If your machine restarts, data capture resumes automatically.
```

Comment:
```suggestion
  If your machine running `viam-server` restarts, data capture resumes automatically.
  (For ESP32 microcontrollers running `viam-micro-server`, unsynced data is lost on restart.)
```

Reply:
That's a bit too detailed at this point. If someone cares about losing data when connection is lost and a device restarts I think they will read more into it

Reply:
agree naomi too in the weeds for this overview of the service

what do yall think about these potential options (solving for Jessamy's concern that we currently don't surface this info anywhere):
i) don't say anything explicit here and explicitly say in the viam-micro-server documentation that unlike viam-server, data capture does not automatically restart.
ii) if you feel it does need to be explicitly said, make a docs pg for behavior on machine restart (as naomi says - if someone cares about losing data when connection is lost and a device restarts, then that pg would be a natural place to look)

Reply:
Oh wait I'm wrong, we do explicitly state this difference on the `capture-sync.md` page in the tabs for viam-server vs viam-micro-server. So I guess I should just remove this part that only applies to viam-server:
```suggestion
```